be/src/common/config.cpp

```diff
@@ -1106,6 +1106,9 @@ DEFINE_mInt64(cache_lock_held_long_tail_threshold_us, "30000000");
 DEFINE_mBool(enable_file_cache_keep_base_compaction_output, "false");
 DEFINE_mInt64(file_cache_remove_block_qps_limit, "1000");
 DEFINE_mInt64(file_cache_background_gc_interval_ms, "100");
+DEFINE_mInt64(file_cache_background_monitor_interval_ms, "5000");
```

Comment:
this config do not used?

Reply:
@cambyzju indeed, I'll delete it in later PR (this is a pick). thanks for pointing that out!